src/AmsConfiguration.cpp

```diff
@@ -603,6 +670,128 @@ bool AmsConfiguration::loadConfig81(int address) {
 	address += readInt(address, &i);
 	setDebugLevel(i);
 
+	bool domo = false;
+	address += readBool(address, &domo);
+
```

Comment:
Remove this, earlier config loading should not be modified.

Reply:
agree, not intensional.

Reply:
done